package.json

```diff
@@ -73,7 +73,6 @@
   },
   "dependencies": {
     "@ember-data/rfc395-data": "^0.0.4",
-    "@glimmer/syntax": "^0.86.0",
```

Comment:
this was unused and was meant to be pulled out earlier -- it _is_ used by the parser tho

Reply:
Would be best to fix in separate PR.

Reply:
done, https://github.com/ember-cli/eslint-plugin-ember/pull/2047